sql_generators/mobile_kpi_support_metrics/README.md

```diff
@@ -8,23 +8,27 @@ This generator aims to provide reusable logic (templates) for generating mobile
 
 Templates included in the generator:
 
-- `attribution_clients` - client level first time attribution in one place
+- `attribution_clients` - client level first time attribution in one place *(expected grain: client_id + normalized_channel)*
 
-- `active_users` - a view on top of baseline_clients_last_seen with some transformations used downstream (acts as the base for the other models)
+- `active_users` - a view on top of baseline_clients_last_seen with some transformations used downstream (acts as the base for the other models) *(expected grain: submission_date + client_id + normalized_channel)*
```

Comment:
including the grain is great!
can we also add population context? 
i.e. 
- population: all clients active on a submission date + clients active in previous 27 days
- population: all clients active on a submission date
- population: all clients